docs/REST API/rest-api.md

```diff
@@ -4141,9 +4185,9 @@ results (i.e., only profiles that are associated with "No team" are listed).
 
 #### Example
 
-Get aggregate status counts of MDM profiles applying to macOS hosts enrolled to Fleet's MDM that are not assigned to any team.
+Get aggregate status counts of MDM profiles applying to macOS and Windows hosts enrolled to Fleet that are not assigned to any team.
```

Comment:
```suggestion
Get aggregate status counts of profiles for to macOS and Windows hosts that are assigned to "No team".
```

Are there other places we can simplify the language like this?

Reply:
Do you mean other places using '**_not assigned to any team_**'? If yes, there are many places using this wording.

Reply:
Ah, ok. Let's wait to make this change in other places then (we can still make it here).

I added an item to product design sync so we can decide and then make a change (if we decide to).

Reply:
@marko-lisica I tweaked my suggestion above to use language like “not assigned to any team.”

I think we can commit the suggestion.